tests/tests/control_iovr4

```diff
@@ -0,0 +1,74 @@
+###############################################################################
+#
+#  global control test: GFSv16 atmosphere only at C96L127 using cloud overlap option iovr=4
+#
+###############################################################################
+
+export TEST_DESCR="Compare global control using iovr=4 results with previous trunk version"
+
+export CNTL_DIR=control_iovr4
+
+export LIST_FILES="sfcf000.nc \
+                   sfcf021.nc \
+                   sfcf024.nc \
+                   atmf000.nc \
+                   atmf021.nc \
+                   atmf024.nc \
+                   GFSFLX.GrbF00 \
+                   GFSFLX.GrbF21 \
+                   GFSFLX.GrbF24 \
+                   GFSPRS.GrbF00 \
+                   GFSPRS.GrbF21 \
+                   GFSPRS.GrbF24 \
+                   RESTART/coupler.res \
```

Comment:
We should stop saving and comparing all these restart files unless this test is going to be used for subsequent restart test(s).